app.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
 require('./index').start();
+
+const mysql = require('mysql');
+const config = require('./config');
+const connection = mysql.createConnection({
```

Comment:
Is this correct?

You are using create pool elsewhere. Surely you are only using one.

Reply:
You're right, I didn't need this. I just checked.  I'll remove it